tests/test_cli_compile.py

```diff
@@ -2779,6 +2779,38 @@ def test_cli_compile_strip_extras(runner, make_package, make_sdist, tmpdir):
     assert "[more]" not in out.stderr
 
 
+def test_cli_compile_all_extras_with_multiple_packages(
+    runner, make_package, make_sdist, tmpdir
+):
+    """
+    Assures that --all-extras works when multiple sources are specified.
```

Comment:
```suggestion
    Assures that `--all-extras` works when multiple sources are specified.
```

Reply:
We should use RST, not Markdown in the docstrings..

Reply:
Well, almost the whole test file uses that (and we don't use autodoc).

Reply:
We should start doing that, then. I usually integrate `sphinxcontrib-apidoc` for automatic generation like this: https://github.com/cherrypy/cheroot/commit/cb9d3f46aa755d63e85d539a2efb7d1fa8f53935. Later on, I remembered to mark the ToC section as **private**: https://cheroot.cherrypy.dev/en/latest/pkg/modules/.

Reply:
Updated now according to the suggestion. I will keep it in line with the rest of the file, if that is okay.